etna/core/mixins.py

```diff
@@ -89,6 +90,49 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    @staticmethod
+    def _update_nested_dict_with_flat_dict(params_dict: dict, flat_dict: dict):
```

Comment:
Can we write a couple of tests on this particular function?

Reply:
Yes, sure